.github/workflows/release.yml

```diff
@@ -20,29 +19,60 @@ jobs:
     - name: get the build-args values
       id: get-build-args
       run: |
-        echo "$(make -s log | grep TAG)" >> "$GITHUB_ENV"
-        echo "$(make -s log | grep GOLANG_VERSION)" >> "$GITHUB_ENV"
+        echo "TAG=v1.29.11-rke2r1-build20241127" >> "$GITHUB_ENV"
+        echo "GOLANG_VERSION=v1.22.8b2" >> "$GITHUB_ENV"
 
-    - name: "Read secrets"
-      uses: rancher-eio/read-vault-secrets@main
-      with:
-        secrets: |
-          secret/data/github/repo/${{ github.repository }}/dockerhub/${{ github.repository_owner }}/credentials username | DOCKER_USERNAME ;
-          secret/data/github/repo/${{ github.repository }}/dockerhub/${{ github.repository_owner }}/credentials password | DOCKER_PASSWORD ;
-          secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials registry | PRIME_REGISTRY ;
-          secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials username | PRIME_REGISTRY_USERNAME ;
-          secret/data/github/repo/${{ github.repository }}/rancher-prime-registry/credentials password | PRIME_REGISTRY_PASSWORD
+    - name: Pre-cleanup disk usage
+      run: |
+        echo "PRE_CLEANUP: $(df -h / | awk 'NR==2 {print $5}' | sed 's/%//')"
+
+    - name: Free up disk space
+      run: |
+        sudo apt-get autoremove -y
+        sudo apt-get clean
+
+        npm cache clean --force
+        yarn cache clean
+
+        docker system prune -af --volumes
+        docker image prune --all --force
+
+        sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+        df -h /
+
+    - name: Pre-build disk usage
+      run: |
+        echo "PRE_BUILD_USAGE=$(df -h / | awk 'NR==2 {print $5}' | sed 's/%//')" >> $GITHUB_ENV
 
     - name: Build and push image
       uses: rancher/ecm-distro-tools/actions/publish-image@master
       with:
         image: hardened-kubernetes
         tag: ${{ github.event.release.tag_name }}
         public-repo: rancher
-        public-username: ${{ env.DOCKER_USERNAME }}
-        public-password: ${{ env.DOCKER_PASSWORD }}
+        public-username: rancher
+        public-password: rancher
```

Comment:
I don't think this should be hard-coded, and the secrets step should be added back